docs/node-operators/index.mdx

```diff
@@ -16,10 +16,10 @@ keywords:
 
 Mina is secured by proof of stake consensus. Unlike other legacy protocols, any participant in a Mina network can validate transactions like a full node — making decentralization possible.
 
-Node operators play two specific roles:
+Node operators are people that run mina nodes that fulfill different roles, including:
 
-1. Block producer - this is analogous to being a Bitcoin "miner" or a "validator" in other proof-of-stake networks. By staking MINA, you can be selected to produce a block and win the block reward.
+1. Block producers - this is analogous to being a Bitcoin "miner" or a "validator" in other proof-of-stake networks. By staking MINA, you can be selected to produce a block and win the block reward.
 
-2. SNARK worker - this job is what helps compress data in Mina's network. The SNARK worker nodes generate proofs of transactions. The block producer buys these proofs on the network (a "snarketplace"). For their efforts, the SNARK worker gets rewarded a part of the block reward.
+2. SNARK workers - this job is what helps compress data in Mina's network. The SNARK worker nodes generate proofs of transactions. The block producer buys these proofs on the network (a "snarketplace"). For their efforts, the SNARK worker gets rewarded a part of the block reward.
```

Comment:
SNARK worker isn't actually a node role. The node role is a SNARK coordinator, which distributes work to a series of SNARK workers in parallel. SNARK workers are just external processes that connect to a Mina node, but are not a first class Mina node themselves.